app/javascript/dashboard/helper/AnalyticsHelper/events.js

```diff
@@ -1,9 +1,69 @@
-export const EXECUTED_A_MACRO = 'Executed a macro';
-export const SENT_MESSAGE = 'Sent a message';
-export const SENT_PRIVATE_NOTE = 'Sent a private note';
-export const INSERTED_A_CANNED_RESPONSE = 'Inserted a canned response';
-export const USED_MENTIONS = 'Used mentions';
-export const MERGED_CONTACTS = 'Used merge contact option';
-export const ADDED_TO_CANNED_RESPONSE = 'Used added to canned response option';
-export const ADDED_A_CUSTOM_ATTRIBUTE = 'Added a custom attribute';
-export const ADDED_AN_INBOX = 'Added an inbox';
+// CONVERSATION EVENTS
+export const CONVERSATION_EVENTS = {
+  EXECUTED_A_MACRO: 'Executed a macro',
+  SENT_MESSAGE: 'Sent a message',
+  SENT_PRIVATE_NOTE: 'Sent a private note',
+  INSERTED_A_CANNED_RESPONSE: 'Inserted a canned response',
+  USED_MENTIONS: 'Used mentions',
+  MERGED_CONTACTS: 'Used merge contact option',
+  ADDED_TO_CANNED_RESPONSE: 'Used added to canned response option',
+  ADDED_A_CUSTOM_ATTRIBUTE: 'Added a custom attribute',
+  ADDED_AN_INBOX: 'Added an inbox',
```

Comment:
Not a conversation action.

Reply:
Move `MERGED_CONTACTS` to the `CONTACTS_EVENTS` category in https://github.com/chatwoot/chatwoot/pull/6234/commits/346707bb2084eb71950e1cc24212abf52e70842b

About the rest, should I create a separate category called `CORE_EVENTS` or `ACCOUNT_EVENTS`?

Reply:
Let us go with ACCOUNT_EVENTS

Reply:
Done